Mobile/ShopLens/Droid/ShopLens.Droid.csproj.user

```diff
@@ -1,8 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SelectedDevice>Android_Accelerated_x86_Oreo</SelectedDevice>
+    <SelectedDevice>LENOVO TB3-850M</SelectedDevice>
```

Comment:
This file shouldn't be commited, I added a gitignore for it in one of my recent branches.